sentences/en/_common.yaml

```
@@ -361,40 +361,40 @@ lists:
     wildcard: true
 
 expansion_rules:
-  name: "[the ]{name}"
-  area: "[the ]{area}"
-  floor: "[the ]{floor}[ floor]"
+  name: "[the] {name}"
+  area: "[the] {area}"
+  floor: "[the] {floor} [floor]"
   area_floor: "(<area>|<floor>)"
-  in_area_floor: "[<in> ]<area_floor>"
-  what_is: "(what's|whats|what is|tell me)[ the]"
-  lockable: "[(the|my) ](lock|door|window|gate|garage door|shutter)[s]"
+  in_area_floor: "[<in>] <area_floor>"
+  what_is: "(what's|whats|what is|tell me) [the]"
+  lockable: "[(the|my)] (lock|door|window|gate|garage door|shutter)[s]"
   where_is: "(where's|wheres|where is)"
-  which: "(which|what)[ of the]"
-  is: "(is|are)[ (there|the)]"
+  which: "(which|what) [of the]"
+  is: "(is|are) [(there|the)]"
   are: "<is>"
-  any: "(any|some)[ of the]"
-  are_any: "[<are> ]<any>"
-  how_many: "how many[ of the]"
-  brightness: "{brightness}[[ ]%| percent]"
-  light: "[the ](light|lights|lighting|lamp|lamps)"
+  any: "(any|some) [of the]"
+  are_any: "[<are>] <any>"
+  how_many: "how many [of the]"
+  brightness: "{brightness} [%|percent]"
+  light: "[the] (light|lights|lighting|lamp|lamps)"
   turn: "(turn|switch|change)"
   temp: "(temp|temperature)"
-  temperature: "{temperature}[([ ]°[ ][{temperature_unit}])|( degrees[ {temperature_unit}])]"
-  open: "(open|raise|lift)[ up]"
-  close: "(close|shut|lower)[ (up|down)]"
+  temperature: "{temperature} [(° [{temperature_unit}])|(degrees [{temperature_unit}])]"
+  open: "(open|raise|lift) [up]"
+  close: "(close|shut|lower) [(up|down)]"
   set: "(set|make|change|turn)"
-  numeric_value_set: "(set|change|turn[ (up|down)]|increase|decrease|make)"
+  numeric_value_set: "(set|change|turn [(up|down)]|increase|decrease|make)"
   in: "(in|on|at|of)"
-  position: "{position}[[ ]%| percent]"
-  volume: "{volume:volume_level}[[ ]%| percent]"
+  position: "{position} [%|percent]"
```

```suggestion
  position: "{position}[ ][%|percent]"
```
We would miss "24%" otherwise